### PR TITLE
items overhaul for 1280_720, credits: Legit#4080

### PR DESCRIPTION
**Pull request by ggodev**
more tests might be needed

### CODE REVIEW

File: params.ini [hunk 1]
```diff
@@ -80,9 +80,12 @@ pathing_delay_factor=5
...
 
 [items]
-;=====MISC=====
+;==================================================================;
+;<<<<<<<<<<<<<<<<<<<<<<<<<<<< MISC >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>;
+;==================================================================;
+
 ;potions
-misc_rejuvenation_potion=1
+misc_rejuvenation_potion=0
```
Comment (aeon0) [inline params.ini:88]: This should be set to 1 by default. Because of this issue #112

File: params.ini [hunk 2]
```diff
@@ -114,141 +117,319 @@ misc_flawless_topaz=0
...
+
+;shields
+magic_monarch=1
+
+;elite paladin shields
+magic_sacred_targe=0
+magic_sacred_rondache=0
+magic_kurast_shield=0
+magic_zakarum_shield=0
+magic_vortex_shield=0
```
Comment (aeon0) [inline params.ini:147]: Any case a magic pala shield is worth anything?

File: params.ini [hunk 3]
```diff
@@ -114,141 +117,319 @@ misc_flawless_topaz=0
...
+magic_ghost_wand=0
+magic_lich_wand=0
+magic_unearthed_wand=0
+
+;assassin katars
+magic_greater_claws=0
+magic_greater_talons=0
+magic_feral_claws=0
+magic_runic_talons=0
+magic_suwayyah=0
```
Comment (aeon0) [inline params.ini:172]: I think shopping blue claws is a lot more effective if you looking for these.

File: params.ini [hunk 4]
```diff
@@ -114,141 +117,319 @@ misc_flawless_topaz=0
...
+magic_wire_fleece=0
+magic_scarab_husk=0
+magic_archon_plate=0
+
+;gloves
+magic_bramble_mitts=0
+magic_vampirebone_gloves=0
+magic_vambraces=0
+magic_crusader_gauntlets=0
+magic_ogre_gauntlets=0
```
Comment (aeon0) [inline params.ini:192]: As with claws, we have a shop bot for these.

File: params.ini [hunk 5]
```diff
@@ -283,82 +464,396 @@ rune_31_jah=2
...
+
+;superior white
+white_superior_archon_plate=1
+white_superior_dusk_shroud=1
+white_superior_light_plate=0
+white_superior_mage_plate=1
+white_superior_great_hauberk=1
+white_superior_wire_fleece=1
+white_superior_scarab_husk=1
+white_superior_wyrmhide=1
```
Comment (aeon0) [inline params.ini:494]: Please put all of these default to 0

**Review by aeon0 — CHANGES_REQUESTED** [review]
Man, so many items : D I dont need all that pickit list, but it seems ppl are asking for it. So thanks to you and Legit! 

Added some comments, will test runtime. If it is still acceptable, we can merge.

File: params.ini (hunk 2 above)
Comment (ggodev) [reply params.ini:147]: jewelers for palas

**Review by aeon0 — APPROVED** [review]

**Pull request merged**